index.html

```diff
@@ -0,0 +1,333 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
+    </header>
+    <main>
+      <div class="intro">
+        <div class="intro-icon">
+          <img src="./media/svg/icon.svg" alt="Icon for the side" />
+        </div>
+        <h2 class="intro-greet">
+          Hey There.
+          <br />
+          I'm Aziz
+        </h2>
+        <h2 class="intro-me">I’am a Software Developer</h2>
+        <p class="intro-text">
+          I can help you build a product, feature or website <br />
+          Look through some of my work and experience! <br />
+          If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <div class="social">
+          <a
+            class="social-link"
+            href="https://github.com/SpaYco"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            <img
+              class="social-icon"
+              src="./media/svg/github.svg"
+              alt="GitHub Icon"
+            />
+          </a>
+          <a
+            class="social-link"
+            href="https://www.linkedin.com/in/spayco"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            <img
+              class="social-icon"
+              src="./media/svg/linkedin.svg"
+              alt="LinkedIn Icon"
+            />
+          </a>
+          <a
+            class="social-link"
+            href="https://angel.co/spayco"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            <img
+              class="social-icon"
+              src="./media/svg/angel.svg"
+              alt="AngelList Icon"
+            />
+          </a>
+          <a
+            class="social-link"
+            href="https://twitter.com/iSpaYco"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            <img
+              class="social-icon"
+              src="./media/svg/twitter.svg"
+              alt="Twitter Icon"
+            />
+          </a>
+          <a
+            class="social-link"
+            href="https://spayco.medium.com/"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            <img
+              class="social-icon"
+              src="./media/svg/medium.svg"
+              alt="Medium Icon"
+            />
+          </a>
+        </div>
+      </div>
+      <div class="projects">
+        <h2>My Recent Work</h2>
+        <hr />
+        <div class="first-card">
+          <img
+            src="./media/png/ubrunning.png"
+            alt="UB-Running Screenshot"
+            class="first-card-img"
```

Comment:
- [ ] Kindly follow the design of the Figma including the colors. The required design should be as the below screenshot.  

![recent works correct design](https://user-images.githubusercontent.com/87413774/163388779-1d27f47e-ee06-4d44-b4eb-ffbbf6f8c52b.png)

- [ ] Here is your design. 

![Recent work wrong design](https://user-images.githubusercontent.com/87413774/163388896-9e8ccf71-7bd6-4aa7-b05c-8457624e7884.png)